examples/dato/README.md

```diff
@@ -0,0 +1,141 @@
+
+## Project Structure
+
+```
+dato/
+├── app/                   # Next.js App Router pages
+│   ├── blog/              # Blog-related routes
+│   ├── api/               # API endpoints
+│   └── [[...path]]/       # Dynamic Makeswift routes
+├── components/            # Reusable React components
+│   └── dato/        # Dato-specific components
+│       ├── common/        # Common scalar components
+│       ├── entries/       # Entry-specific components
+│       │   └── BlogPost/  # BlogPost entry components
+│       └── queries/       # GraphQL queries
+├── lib/                   # Utility functions and configs
+└── generated/             # Auto-generated types from Dato
+```
+
+## Quick Start
+
+### 1. Set up the project
+
+   ```bash
+   npx makeswift@latest init --example=dato
+   ```
+
+### 2. Configure environment variables in `.env.local`
+
+   ```
+   MAKESWIFT_SITE_API_KEY=your_makeswift_api_key
+   DATO_CMS_API_TOKEN=your_dato_api_token
+   ```
+
+### 3. Set up your content model
+
+To get started with blog posts, you'll need to set up a content model in Dato. Here's what we recommend:
+
+- Create a new content type called "Author" with the following fields:
+
+   - **Name** (Short text) - The name of the author. Make sure to check "This field represents the Entry title" in the field options.
+   - **Slug** (Short text) - A URL-friendly identifier (e.g. "author-name")
+   - **Job Title** (Short text) - The author's job title
+   - **Description** (Rich text) - The main content as rich text
+
+- Create a new content type called "BlogPost" with the following fields:
+
+   - **Title** (Single-line text) - A short text field for the post title. Make sure to check "This field represents the Entry title" in the field options.
+   - **Description** (Multi-paragraph text) - A brief summary of the content
+   - **Slug** (Single-line text) - A URL-friendly identifier (e.g. "my-first-blog-post")
+   - **Feed Date** (Date) - When the post should appear in feed
+   - **Body** (Structured text) - The main content as rich text
+   - **Author** (Link to Author) - Which Author created the blog. Ensure that "Accept only specified entry type" is enabled in order to introspect this reference.
+   - **Banner** (Image) - An optional hero image for the post
+   
+Be sure to add a few blog posts for testing purposes, and connect them to an Author.
+
+### 4. Generate Dato types
+
+   ```bash
+   pnpm codegen-ts
+   ```
+
+Note that the only query we are working with is in `/components/dato/GetBlogs.graphql`.
+
+### 5. Run the development server
+   ```bash
+   pnpm dev
+   ```
+
+## Building Blog Post pages in Makeswift
+
+1. Publish your content in Dato.
+2. In the [Makeswift builder](https://docs.makeswift.com/product/builder-basics), navigate to your blog post page by entering the post's URL (e.g., `/blog/welcome`) in the builder's URL bar.
```

Comment:
I'm not sure that link makes sense? i would expect that link to take me to the builder directly